regulations/templates/regulations/base.html

```diff
@@ -11,10 +11,12 @@
     <!-- Global site tag (gtag.js) - Google Analytics -->
     <script async src="https://www.googletagmanager.com/gtag/js?id={{ GA_ID }}"></script>
     <script>
+        if ( "{{ AUTOMATED_TEST }}" === "true" ) {
+            window["ga-disable-{{ GA_ID }}"] = true;
```

Comment:
Should be able to simplify this to `window["ga-disable-{{ GA_ID }}"] = {{ AUTOMATED_TEST }}`

Reply:
yeah good call - I ended up with this: 

`window["ga-disable-{{ GA_ID }}"] = {{ AUTOMATED_TEST | yesno:"true,false" }};`